shaq/_cli.py

```diff
@@ -178,7 +178,8 @@ def main() -> None:
             sys.exit(1)
 
         try:
-            raw = asyncio.run(_shaq(console, args))
+            loop = asyncio.get_event_loop()
+            raw = loop.run_until_complete(_shaq(console, args))
```

Comment:
Could you explain how this addresses the "Event loop is closed" error you're seeing? I haven't seen that locally, and my understanding is that `run(...)` is considered more idiomatic for this type of single coro.

Reply:
It's Windows problem and related to ProactorEventLoop, which is default on Windows since Python >=3.8. Full console output:

```
Track: Demon Or Two (feat. LEECHY!)
Artist: HalaCG
Album Cover: https://is1-ssl.mzstatic.com/image/thumb/Music122/v4/4f/98/67/4f9867f8-f6e6-044b-578f-0601b0ee7549/artwork.jpg/1000x1000cc.png
Exception ignored in: <function _ProactorBasePipeTransport.__del__ at 0x00000000032C71C0>
Traceback (most recent call last):
  File "C:\Users\Hiro\.pyenv\pyenv-win\versions\3.10.5\lib\asyncio\proactor_events.py", line 116, in __del__
    self.close()
  File "C:\Users\Hiro\.pyenv\pyenv-win\versions\3.10.5\lib\asyncio\proactor_events.py", line 108, in close
    self._loop.call_soon(self._call_connection_lost, None)
  File "C:\Users\Hiro\.pyenv\pyenv-win\versions\3.10.5\lib\asyncio\base_events.py", line 750, in call_soon
    self._check_closed()
  File "C:\Users\Hiro\.pyenv\pyenv-win\versions\3.10.5\lib\asyncio\base_events.py", line 515, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
```

There are also one other way of fixing this. 

### Force SelectorEventLoop Policy

```python
if os.name == "nt":
    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy)
```

Should I stick to this instead?

Reply:
Yeah, I think that would be slightly simpler, thank you!